Appraisals

```diff
@@ -21,3 +21,7 @@ end
 appraise "rails-52" do
   gem "rails", "~> 5.2.0"
 end
+
+appraise "rails-60" do
```

Comment:
I think after adding this you need to run `appraisal generate  ` to generate a gemfile with rails 6

Reply:
Just added, thank you!